src/main/java/util/GitHubURL.java

```diff
@@ -49,7 +49,7 @@ public static boolean isUrlIssue(String url) {
     }
     
     public static boolean isPullRequestLoaded(String url)  {
-        return (url.endsWith("/commits") || url.endsWith("/files"));
+        return url.matches("https://github.com/([^/]+)/([^/]+)/pull/([0-9]+)([/commits,/files]*)");
```

Comment:
The last part doesn't look right. `[/commits,/files]*` is a character class, so it will match things like `cc`, `,,` -- 0 or more of any character in the brackets.

Also we don't have to parenthesise groups that we're not going to refer to. That usually makes the regex a tiny bit faster.

This should probably have tests, given how mistakes in regexes can sometimes be very subtle.